Sources/Foundation/Process.swift

```diff
@@ -1149,7 +1149,11 @@ open class Process: NSObject, @unchecked Sendable {
     // convenience; create and launch
     open class func launchedProcess(launchPath path: String, arguments: [String]) -> Process {
         let process = Process()
-        process.launchPath = path
+        if #available(macOS 10.13, *) {
```

Comment:
With our new approach to swift-corelibs-foundation, it no longer builds for macOS. So, I think in the end this ifdef is not required - we can just always use the URL version.